root/search/components/InstrumentResults.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../../context';
+import {l} from '../../static/scripts/common/i18n';
+import {lp_attributes} from '../../static/scripts/common/i18n/attributes';
+import EntityLink from '../../static/scripts/common/components/EntityLink';
+import loopParity from '../../utility/loopParity';
+import type {ResultsPropsT} from '../types';
+
+import PaginatedSearchResults from './PaginatedSearchResults';
+import ResultsLayout from './ResultsLayout';
+
+function buildResult(result, index) {
+  const instrument = result.entity;
+  const score = result.score;
+
+  return (
+    <tr className={loopParity(index)} data-score={score} key={instrument.id}>
+      <td>
+        <EntityLink entity={instrument} />
+      </td>
+      <td>{instrument.typeName ? lp_attributes(instrument.typeName, 'instrument_type') : null}</td>
+      <td>{instrument.description}</td>
```

Comment:
Some of the descriptions contain HTML, so you'll have to tell React by using `dangerouslySetInnerHTML` on the `<td>` (otherwise it'll display HTML to the user, since it escapes everything by default).

See how https://github.com/metabrainz/musicbrainz-server/blob/master/root/instrument/List.js#L35-L35 does it. Note it also uses `l_instrument_descriptions` for translation.